cms/static/js/features_jsx/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
@cahrens @andy-armstrong the course_experience eslintrc isn't working here because it doesn't contain any configuration for React. eslint does not parse jsx natively and [recommends extending eslint-config-airbnb](http://eslint.org/docs/user-guide/configuring#specifying-parser-options) to enable jsx support. This can be done directly within this particular eslintrc, but it's better to add it to [eslint-config-edx](https://github.com/edx/eslint-config-edx) instead so it can be used globally.